spec/models/job_application_spec.rb

```diff
@@ -24,6 +24,18 @@
     end
   end
 
+  context "when setting support needed to 'no'" do
+    subject { create(:job_application) }
```

Comment:
put some explicit value here for `support_needed_details` so it is clear that it is going to be blanked

Reply:
Sorry yeah that's obvious isn't it lol